packages/core/CalendarManager.ts

```diff
@@ -253,9 +254,7 @@ export const createEvent = async (
           calError = error.calError;
         }
         log.error("createEvent failed", error, calEvent);
-        // @TODO: This code will be off till we can investigate an error with it
-        //https://github.com/calcom/cal.com/issues/3949
```

Comment:
We should identify if the broken integration email is still sent for successful calendar connection as well.

Reply:
So the reason we disabled this was it was sending false negatives on production but we couldn't replicate the issue locally. I would be hesitant.

Reply:
I wouldn't be opposed to this idea.

Reply:
Agreed!

Reply:
Reopening this PR and we need to handle this as well https://github.com/calcom/cal.com/pull/11702